src/components/LinkTag/LinkTag.tsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+
+interface AGroup extends React.AnchorHTMLAttributes<HTMLAnchorElement> {}
+export const LinkTag: React.ComponentType<AGroup> = ({ href, className, children, ...rest }) => (
+    <a href={href} className={className} {...rest}>
+        {children}
+    </a>
+);
```

Comment:
ok, nice but could be better:
```suggestion
export const LinkTag = ({ href, className, children, ...rest }: React.ComponentProps<"a">) => (
    <a href={href} className={className} {...rest}>
        {children}
    </a>
);
```